docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -0,0 +1,288 @@
+description: Restore data from a specific point in time in YugabyteDB for YCQL
+beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+aliases:
+menu:
+  latest:
+    identifier: point-in-time-restore-2-ycql
+    parent: backup-restore
+    weight: 704
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="/latest/manage/backup-restore/point-in-time-restore-ysql" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+  <li >
+    <a href="/latest/manage/backup-restore/point-in-time-restore-ycql" class="nav-link active">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+{{< note>}}
+Refer to the [YSQL tab](../point-in-time-restore-ysql) for details on this feature.
+{{</ note >}}
+
+## Try out the PITR feature
+
+You can test the PITR feature (BETA) by creating a database and populating it, creating a snapshot, and restoring ([data only](#limitations)!) from that snapshot.
+
+### Create and snapshot a table
+
+Create and populate a table, look at a timestamp to which you'll restore, and then write a row.
+
+1. Start the YCQL shell and connect to your local instance:
+
+    ```sh
+    $ bin/ycqlsh
+    ```
+
+1. Create a table and populate some sample data:
+
+    ```sql
+    create keyspace pitr;
+
+    use pitr;
+
+    create table employees (
+      employee_no integer PRIMARY KEY,
+      name text,
+      department text,
+      salary integer
+    );
+
+    insert into employees (employee_no, name, department, salary) values (1221, 'John Smith', 'Marketing', 50000);
+    insert into employees (employee_no, name, department, salary) values (1222, 'Bette Davis', 'Sales', 55000);
+    insert into employees (employee_no, name, department, salary) values (1223, 'Lucille Ball', 'Operations', 70000);
+    insert into employees (employee_no, name, department, salary) values (1224, 'John Zimmerman', 'Sales', 60000);
+
+    SELECT * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+1. Create a snapshot of the table from a shell prompt:
+
+    ```sh
+    $ bin/yb-admin create_database_snapshot pitr employees
+    ```
+
+    ```output
+    Started snapshot creation: bb5fc435-a2b9-4f3a-a510-0bacc6aebccf
+    ```
+
+1. Verify that the snapshot is complete:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    No snapshot restorations
+    ```
+
+### Restore from an absolute time
```

Comment:
For the absolute time, we should also add a link to https://docs.yugabyte.com/latest/api/ycql/type_datetime/#timestamp , because the restore_timestamp also takes that string input.

Reply:
(Done)